draft-ietf-rats-eat.md

```diff
@@ -1786,6 +1800,54 @@ quality random number generators are readily available as they are
 implemented in commonly used CPU hardware.
 
 
+# EAT Relation to IDevID
+
+This section describes several distinct ways in which an IEEE IDevID [IDevID] relates to EAT, particularly to UEID and SUEID.
+
+
+## Signed Device IDs
+
+A unique device identifier is of little use in security-related applications unless it is signed.
+Without signing, it is trivial for attackers to forge a device identifier.
+
+In a way IDevID and EAT are competitors.
+They both provide signed unique device identifiers.
+
+EAT provides the unique identifier as a single small data item, the UEID and SUEID.
+For IDevID the unique identifier is through and X.500 distinguished name, perhaps constructed from the certificate chain, 
+some how derived from the key material.
+
+This document argues EAT is the more general solution.
```

Comment:
I suggest to drop this, it sounds like a value judgement.  Just say "Compared to DevID, EAT allows an open-ended [...]"

Reply:
Have done a lot of rewording do better here.